models/classes/import/class.TestImportForm.php

```diff
@@ -132,4 +135,17 @@ private function addMetadataImportElement()
             $this->form->addToGroup('file', self::METADATA_FORM_ELEMENT_NAME);
         }
     }
+
+    private function addItemDestinationPlacementComponent(): void
+    {
+        if (!$this->isFieldDisabled(self::ITEM_CLASS_DESTINATION_FIELD)) {
+            $selectElt = tao_helpers_form_FormFactory::getElement('selectelt', 'Free');
+            $selectElt->setValue('<div class="item-select-container"></div>');
+
+            $itemClassDestination = tao_helpers_form_FormFactory::getElement(self::ITEM_CLASS_DESTINATION_FIELD, 'Hidden');
```

Comment:
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
--------------------------------------------------------------------------------
 145 | WARNING | Line exceeds 120 characters; contains 123 characters